src/precompiles.rs

```diff
@@ -1,8 +1,22 @@
+use crate::parameters::BridgedTokenWithdrawEthConnectorArgs;
 use crate::prelude::{Address, Borrowed, Vec, H160, H256, U256};
 use evm::{Context, ExitError, ExitSucceed};
 
+use alloc::string::{String, ToString};
+use borsh::BorshSerialize;
+
+use crate::parameters::{BridgedTokenWithdrawArgs, NEP41TransferCallArgs};
+use crate::sdk;
+use crate::types::{AccountId, Gas};
+
 type PrecompileResult = Result<(ExitSucceed, Vec<u8>, u64), ExitError>;
 
+// Computed as: near_account_to_evm_address("exitToEthereum".as_bytes()).to_low_u64_be()
+const EXIT_TO_ETHEREUM_ID: u64 = 17176159495920586411;
+
+// Computed as: near_account_to_evm_address("exitToNear".as_bytes()).to_low_u64_be()
+const EXIT_TO_NEAR_ID: u64 = 11421322804619973199;
+
 #[allow(dead_code)]
 pub fn no_precompiles(
```

Comment:
Maybe this should be renamed to `no_externel_precompiles` or something along those lines?